Source/Core/DolphinQt2/DolphinQt2.vcxproj

```diff
@@ -91,6 +91,8 @@
     <QtMoc Include="Settings\AudioPane.h" />
     <QtMoc Include="MainWindow.h" />
     <QtMoc Include="MenuBar.h" />
+    <QtMoc Include="NetPlay/NetPlayDialog.h" />
+    <QtMoc Include="NetPlay/NetPlaySetupDialog.h" />
```

Comment:
Looks a little inconsistent with only this one having forward slashes instead of back slashes...